src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +193,20 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface Trigger{
+    val id: String
+    val type: String
+
+    fun fetchProperties(): List<Parameter>
+}
+
+interface ArtifactDependency{
+    val id: String
+    val type: String
+    val disabled: Boolean
+    val inherited: Boolean
+    val sourceBuildType: BuildType
```

Comment:
You shouldn't blindly copy name of attributes from json to API. We have separate API and *Bean classes exactly to allow providing comprehensible names in API.